content/en/docs/concepts/policy/validating-admission-policy/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: ValidatingAdmissionPolicy
+content_type: concept
+weight: 30
+description: Validating admission policies allow configurable checks in the API server using the Common Expression Language (CEL). For example, a `ValidatingAdmissionPolicy` can be used to disallow use of the `latest` image tag. [More...](/docs/concepts/policy/validating-admission-policy/)
```

Comment:
Don't put Markdown in descriptions (it doesn't render everywhere you might want it to)